### PR TITLE
clean: Refactor information architecture DOCS-155

### PR DESCRIPTION
**Pull request by pauloribeiro-codacy**
The main factor driving this Information Architecture reorganization was the fact that we had several pages under the section **Your repositories** that make more sense under **Configuring your repositories**. These two sections have different audiences:

- The audience for the pages under **Your repositories** are people with Read or Write permissions;
- While most actions under **Configuring your repositories** can only be done by people with Admin permissions.

[This slide deck](https://docs.google.com/presentation/d/16P4EDHhO7Wt3FnwAJxeqJ2T-SJipz6Ou3YOf9TDtoh4/edit) presents all the details of the change in a clear way so it's easier to provide feedback.

- [x] Move files to their correct folders and update all links (:warning: don't forget the homepage)
- [ ] Review links from codacy-website and codacy-spa to documentation
    - I will do this task separately (outside this pull request)

### CODE REVIEW

File: docs/getting-started/getting-started-with-codacy.md [hunk 1]
```diff
@@ -25,23 +23,29 @@ Now, you'll need to join one or more organizations that contain your repositorie
 
 To start adding your repositories, select one of the organizations and click **Go to add repositories**.
 
-## 4. Add repositories {: id="add-repositories"}
+## 3. Add repositories {: id="add-repositories"}
 
 As a final step, add the repositories that you wish to analyze. Codacy will start the first analysis and set up everything required to ensure your next commits on those repositories are analyzed.
```
Comment (inesbpinto) [inline docs/getting-started/getting-started-with-codacy.md:28]: I know the scope of this PR is around IA *but* would it be worth mentioning here that only repo admins can add repositories? (I'd confirm the truthfulness of this for Bitbucket)

File: docs/getting-started/getting-started-with-codacy.md [hunk 2]
```diff
@@ -25,23 +23,29 @@ Now, you'll need to join one or more organizations that contain your repositorie
...
-## You're all set! 🎉 {: id="all-set"}
-
 Codacy begins an initial analysis as soon as you add a repository, and displays an overview of the code quality of your repository when the analysis is complete.
 
-After that, you can continue to explore and configure Codacy for your repository:
+![Repository dashboard](../repositories/images/repository-dashboard.png)
+
+## 4. Tweak your repositories
+
+ [Check the static analysis results](../repositories/issues-view.md) on the Issues page to explore the initial analysis results, and tweak the Codacy configurations for your repositories:
```
Comment (inesbpinto) [inline docs/getting-started/getting-started-with-codacy.md:38]: (again, scope but here it goes anyway, Paulo 😂) Maybe would be cool to mention that this initial analysis is based on Codacy defaults and encourage users to tweak the patterns.

File: docs/getting-started/getting-started-with-codacy.md (hunk 1 above)
Comment (pauloribeiro-codacy) [reply docs/getting-started/getting-started-with-codacy.md:28]: Hello @inesbpinto! :wave: :smile: 

Don't worry, this pull request got a bit more complicated than what I was expecting, and I'll probably have to start from scratch after I finish making some other unrelated changes that also affect the IA a bit. :+1: 

Your suggestions do make a lot of sense! :raised_hands: I opened this issue #418 on your behalf and will tackle it very soon. :+1:

Comment (pauloribeiro-codacy) [reply docs/getting-started/getting-started-with-codacy.md:28]: @inesbpinto Just as a heads up, I opened a [separate pull request](https://github.com/codacy/docs/pull/428) to make the improvements to the "Getting started", separate from all the other IA changes which are a bit stuck at the moment. :)

**Review by IGFCoimbra — DISMISSED** [review]

**Pull request merged**